app/routes/events/components/EventAdministrate/RegistrationTables.js

```diff
@@ -293,6 +293,13 @@ export class RegisteredTable extends Component<Props> {
             handlePayment={handlePayment}
           />
         ),
+        sorter: (a, b) => {
+          const paymentStatusA = a.paymentStatus ?? 'failed';
+          const paymentStatusB = b.paymentStatus ?? 'failed';
+
+          if (paymentStatusA > paymentStatusB) return 1;
+          else return -1;
```

Comment:
This could be simplified a bit with [Logical nullish assignments (??=)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment) and [Ternary operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

```suggestion
          a.paymentStatus ??= 'failed';
          b.paymentStatus ??= 'failed';

          return a.paymentStatus > b.paymentStatus ? 1 : -1;
```

Reply:
As we do not support the logical nullish assignment syntax, I only changed to ternary operators